pkg/notifier/notifier.go

```diff
@@ -47,7 +47,7 @@ type Notifier struct {
 }
 
 type Info struct {
-	event EventType
+	Event EventType
```

Comment:
Maybe it'd be better make `Event` an exported field already in the patch where we introduce notifier package